server.js

```diff
@@ -85,21 +85,28 @@ app.use(express.static(path.join(__dirname, 'public')));
 app.get(pathPrefix + 'login', login.get);
 app.post(pathPrefix + 'login', jsonParser, login.post);
 
-app.all(appPrefix + 'update_settings/*', function(req, res) {
-  // don't audit the app settings
-  proxy.web(req, res);
-});
-app.all(pathPrefix + '_revs_diff', function(req, res) {
-  // don't audit the _revs_diff
-  proxy.web(req, res);
-});
-app.all(pathPrefix + '_local/*', function(req, res) {
-  // don't audit the _local docs
-  proxy.web(req, res);
-});
-app.all(pathPrefix + '_bulk_get/*', function(req, res) {
-  // don't audit the _bulk_get request
-  proxy.web(req, res);
+// Requests we don't wish to audit
+// (usually because POST is just used here to pass more parameters)
+[
+  appPrefix + 'update_settings/*',
```

Comment:
Auditing works by taking the given doc so we can get the ID and therefore find the audit doc later. The update_settings endpoint doesn't take a doc it just takes a JSON string with no ID. We could audit it but we'd need to think about how to do that (eg: inject the right ID?). But the bottom line is the existing audit framework can't handle objects without IDs.